notebooks/contributing/formatting.md

```diff
@@ -0,0 +1,289 @@
+Tutorial Structure and Formatting Requirements
+==============================================
+
+To ensure quality and consistency, the [Jupyter Notebooks](https://jupyter.org/) containing tutorials must follow strict structure and formatting requirements. The contents of each notebooks must be parsed sensibly into a set of sections. Sufficient explanation and links to API documentation are required when presenting functionality. Each section within a tutorial contains a combination of [Markdown](https://www.markdownguide.org/getting-started/) and coding cells. Our notebooks use the functionality in [MyST Parser](https://myst-parser.readthedocs.io/en/latest/) to augment the contents of Markdown cells. Before moving forward, we advise the contributor to:
+
+* Be familiar with the structure and formatting that is used by most forward simulation and inversion tutorials. E.g. [3D Forward Simulation of Gravity Anomaly Data](../03-gravity/fwd_gravity_anomaly_3d.ipynb) and [3D Inversion of Gravity Anomaly Data](../03-gravity/inv_gravity_anomaly_3d.ipynb).
+
+* Refer to the Markdown syntax used by [MyST Parser](https://myst-parser.readthedocs.io/en/latest/).
+
+* Refer to the [style guide](https://docs.simpeg.xyz/latest/content/getting_started/contributing/code-style.html) for coding cells.
+
+
+## Notebook Introduction
+
+Every tutorial notebook requires an introduction section. The Markdown cells required for the introduction are described below.
+
+### Cell 1: Title Cell
+
+The first cell contains the title and the author for the tutorial. It is formatted as follows.
+The title is in quotations. You will need to add yourself as an author in the `myst.yml` file in the root directory of the repository.
+
+```
+---
+title: "My Notebook Title In Quotes"
+authors:
+  - id: devincowan
+---
+```
```

Comment:
We can use code highlighting to make this look prettier:
```suggestion
```yaml
---
title: "My Notebook Title In Quotes"
authors:
  - id: devincowan
---
```
```